### PR TITLE
Do not assert on invalid GLO frequency slot

### PR DESCRIPTION
**Pull request by IsakTjernberg**
When receiving an GLN ephemeris for sat id < 1 or sat id > 28 we would assert, with this PR we simply return instead.
Also see [https://snav.slack.com/archives/CDBCP43BM/p1602244115191100](https://snav.slack.com/archives/CDBCP43BM/p1602244115191100).

### CODE REVIEW

File: src/glo_map.c [hunk 1]
```diff
@@ -67,8 +67,9 @@ bool glo_map_valid(const gnss_signal_t sid) {
  * @param[in] glo_slot_id GLO orbital slot
  */
 void glo_map_set_slot_id(u16 fcn, u16 glo_slot_id) {
-  assert(glo_slot_id_is_valid(glo_slot_id));
-  assert(glo_fcn_is_valid(fcn));
+  if (!glo_slot_id_is_valid(glo_slot_id) || !glo_fcn_is_valid(fcn)) {
+    return;
```
Comment (jbangelo) [inline src/glo_map.c:71]: Might be useful to add a debug log here saying that this failed.

**Review by jbangelo — APPROVED** [review]

File: src/glo_map.c (hunk 1 above)
Comment (pgrgich) [reply src/glo_map.c:71]: Agreed.

**Review by pgrgich — APPROVED** [review]
Lgtm.

**Review by sasamb — APPROVED** [review]
Lgtm, thanks!

**Pull request merged**